framework/src/timeintegrators/ExplicitTimeIntegrator.C

```diff
@@ -45,11 +45,11 @@ ExplicitTimeIntegrator::validParams()
 ExplicitTimeIntegrator::ExplicitTimeIntegrator(const InputParameters & parameters)
   : TimeIntegrator(parameters),
     MeshChangedInterface(parameters),
-
     _solve_type(getParam<MooseEnum>("solve_type")),
     _explicit_residual(addVector("explicit_residual", false, PARALLEL)),
     _solution_update(addVector("solution_update", true, PARALLEL)),
-    _mass_matrix_diag_inverted(addVector("mass_matrix_diag_inverted", true, GHOSTED))
+    _mass_matrix_diag_inverted(addVector("mass_matrix_diag_inverted", true, GHOSTED)),
+    _ones(nullptr)
```

Comment:
By the way, `_ones` ends up being reset to a nullptr when we enter meshChanged.

Reply:
Turns out there is a second time integrator that get's called on meshChanged, which has _ones set to zero (probably because it has `_nl` set to 0). I wonder where that one is coming from?